code/modules/surgery/gastrectomy.dm

```diff
@@ -0,0 +1,48 @@
+/datum/surgery/gastrectomy
+	name = "Gastrectomy"
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_real_bodypart = TRUE
+	steps = list(/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/saw,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/incise,
+		/datum/surgery_step/gastrectomy,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/close
+		)
+
+/datum/surgery/gastrectomy/can_start(mob/user, mob/living/carbon/target)
+	var/obj/item/organ/liver/L = target.getorganslot(ORGAN_SLOT_STOMACH)
```

Comment:
```suggestion
	var/obj/item/organ/L = target.getorganslot(ORGAN_SLOT_STOMACH)
```

Reply:
Remind me to never code at 4am and sleep deprived, or type in words I was thinking minutes ago